### DISCUSSION TITLE
How to overwrite the link overlay panel for TinyMCE Editor?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17330] [Archive]
Is it possible to change this link modal that comes with your WYSIWYG editor?

<img width="843" alt="screenshot" src="https://user-images.githubusercontent.com/2291224/215273537-720022bc-85e3-4662-9551-f97f5925cfb5.png">

I need to completely customize it …

**Comment by rijkvanzanten** [comment]
There's no way to overwrite just that modal but keep the rest of the interface as core. A new custom interface would be the way to customize it fully.
